temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1081 @@
+```
 
 <!--
 * maximum-args=1
 -->
 
 #### Options
 
-* `--namespace-id` (string) -  Namespace ID.
+* `--namespace-id` (string) -
+  Namespace ID.
 
-### temporal operator namespace list:  List all Namespaces.
+### temporal operator namespace list: List Namespaces
 
-The temporal operator namespace list command lists all Namespaces on the Server.
+Display a detailed listing for all Namespaces on the Service:
 
-### temporal operator namespace update: Updates a Namespace.
+```
+temporal operator namespace list
+``` 
 
-The temporal operator namespace update command updates a Namespace.
+### temporal operator namespace update: Update a Namespace
 
-Namespaces can be assigned a different active Cluster.
-`temporal operator namespace update -n namespace --active-cluster=NewActiveCluster`
+Update a Namespace using properties you specify.
 
-Namespaces can also be promoted to global Namespaces.
-`temporal operator namespace update -n namespace --promote-global`
+```
+temporal operator namespace update [options]
+``` 
 
-Any Archives that were previously enabled or disabled can be changed through this command.
-However, URI values for archival states cannot be changed after the states are enabled.
-`temporal operator namespace update -n namespace --history-archival-state=enabled --visibility-archival-state=disabled`
+Assign a Namespace's active Cluster (Service):
+
+```
+temporal operator namespace update \
+    --namespace YourNamespaceName \
+    --active-cluster NewActiveCluster
+```
+
+Promote a Namespace for multi-region data replication:
+
+```
+temporal operator namespace update \
+    --namespace YourNamespaceName \
+    --promote-global
+```
+
+You may update archives that were previously enabled or disabled. Note: URI
+values for archival states can't be changed once enabled.
+
+```
+temporal operator namespace update \
+    --namespace YourNamespaceName \
+    --history-archival-state enabled \
+    --visibility-archival-state disabled
+```
 
 <!--
 * maximum-args=1
 -->
 
 #### Options
-* `--active-cluster` (string) - Active cluster name.
-* `--cluster` (string[]) - Cluster names.
-* `--data` (string[]) - Namespace data in key=value format. Use JSON for values.
-* `--description` (string) - Namespace description.
-* `--email` (string) - Owner email.
-* `--promote-global` (bool) - Promote local namespace to global namespace.
-* `--history-archival-state` (string-enum) - History archival state. Options: disabled, enabled.
-* `--history-uri` (string) - Optionally specify history archival URI (cannot be changed after first time archival is enabled).
-* `--retention` (duration) - Length of time a closed Workflow is preserved before deletion.
-* `--visibility-archival-state` (string-enum) - Visibility archival state. Options: disabled, enabled.
-* `--visibility-uri` (string) - Optionally specify visibility archival URI (cannot be changed after first time archival is enabled).
+* `--active-cluster` (string) -
+  Active Cluster (S) name.
```

Comment:
What does the `(S)` here mean?

Reply:
(Service). Not sure why it got de-erviced.